sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/listeners/OnCameraChangeListener.kt

```diff
@@ -14,6 +15,8 @@ fun interface OnCameraChangeListener {
    *
    * For more precise information about actual map camera animations
    * please check [CameraAnimationsPlugin], [CameraAnimationsLifecycleListener] and [CameraAnimatorChangeListener]
+   *
+   * @param eventData CameraChangedEventData
    */
-  fun onCameraChanged()
+  fun onCameraChanged(eventData: CameraChangedEventData)
```

Comment:
just curious: will it break all existing Java code using our SDK?

Reply:
yes, it will 🙃